java/src/main/java/vowpalWabbit/learner/VWLearners.java

```diff
@@ -36,6 +36,8 @@ private VWLearners() {}
      */
     @SuppressWarnings("unchecked")
     public static <T extends VWLearner> T create(final String command) {
+        if(command.indexOf("--no_stdin") == -1)
+            command += " --no_stdin";
```

Comment:
Travis build failure: `/vw/java/src/main/java/vowpalWabbit/learner/VWLearners.java:[40,13] final parameter command may not be assigned`

`command` is final so can't be changed, can just create a new string to append to to fix.